### PR TITLE
CSV download (again)

### PR DESCRIPTION
**Pull request by fisx**
More left-overs from #1351 #1355:

- [ ] drop check for accept header.
- [ ] auth by cookie, not access token.
- [ ] make sure nginz has the route open and unauthenticated everywhere.
- [ ] add scim externalId or saml nameID.
- [ ] add rich info.

### CODE REVIEW

File: services/galley/src/Galley/API/Public.hs [hunk 1]
```diff
@@ -177,7 +177,6 @@ sitemap = do
     -- nginz metrics dashboards easier.
     zauthUserId
       .&. capture "tid"
-      .&. accept "text" "csv"
```
Comment (fisx) [inline services/galley/src/Galley/API/Public.hs:180]: @Yserz you won't set the accept header in the csv download request, right?  or will you?

Comment (Yserz) [reply services/galley/src/Galley/API/Public.hs:180]: With an anchor tag I can not modify any headers. The browser would add every mime-type to the header that is acceptable for the browser.

Comment (fisx) [reply services/galley/src/Galley/API/Public.hs:180]: Ok, let's remove it then, seems safer to me.  Thanks!

**Comment by smatting** [comment]
Closing this because we decided not to do auth by cookie. [context](https://wearezeta.atlassian.net/browse/SQSERVICES-107?focusedCommentId=94058)

**Pull request closed without merging**